File: CursoPOOUber/Java/Tarjeta.java
package Repositorio_POO.CursoPOOUber.Java;


class Tarjeta extends Payment{
    String numbers;
    String date;
    Integer cvv;
    
    public Tarjeta(int id, String numbers, String date, Integer cvv){
        super(id);
        this.numbers = numbers;
        this.date = date;
        this.cvv = cvv;
    }
    
}

File: CursoPOOUber/Python/tarjeta.py
import numbers
from payment import Payment;

class Tarjeta(Payment):
    numbers = str;
    date = str;
    cvv = str;

    def __init__(self, id, numbers, date, cvv):
        super().__init__(id)
        self.numbers =  numbers
        self.date = date
        self.cvv = cvv